DayTomato/DayTomato/Services/DayTomatoClient.cs

```diff
@@ -51,11 +71,89 @@ public DayTomatoClient()
 			return false;
 		}
 
+		// Delete Pin
+		public async Task<bool> DeletePin(Pin pin)
+		{
+			var uri = new Uri(BASE_URL + "/api/pins/" + pin.Id);
+			var response = await httpClient.DeleteAsync(uri);
+			if (response.IsSuccessStatusCode)
+			{
+				return true;
```

Comment:
Is there future functionality we might be using in these ifs? If not, there could be a couple of ways we can collapse this down:

1. Reduce it to `return response.IsSuccessStatusCode;`.
2. Change the method return type to `Task` instead of `Task<T>` to get a "async void" equivalent.
3. Throw appropriate exceptions for the different status codes that might be returned.

Thoughts?